test/reputation-update.js

```diff
@@ -271,20 +275,26 @@ contract("Reputation Updates", accounts => {
     });
 
     it("should calculate nUpdates correctly when making a log", async () => {
-      await metaColony.addGlobalSkill(1);
-      await metaColony.addGlobalSkill(4);
-      await metaColony.addGlobalSkill(5);
-      await metaColony.addGlobalSkill(6);
-
-      await setupFinalizedTask({ colonyNetwork, colony: metaColony, skillId: 5 });
-      let repLogEntryWorker = await inactiveReputationMiningCycle.getReputationUpdateLogEntry(4);
-      expect(repLogEntryWorker.amount).to.eq.BN(WORKER_PAYOUT);
+      // Replace addDomain with the addDomain implementation with no restrictions on depth of subdomains
+      const noLimitSubdomains = await NoLimitSubdomains.new();
+      const resolverAddress = await colonyNetwork.getColonyVersionResolver(1);
+      const resolver = await Resolver.at(resolverAddress);
+      await resolver.register("addDomain(uint256)", noLimitSubdomains.address);
```

Comment:
I'm seeing this code block a lot -- breaking this out into a test helper would make this PR slightly more succinct.